src/core/tests/library/LegacyEnvLayerExtensions.py

```diff
@@ -538,7 +540,10 @@ def run_command_output(self, cmd, no_output=False, chk_err=True):
                              "model name	: Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz\n" + \
                              "model name	: Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz"
                 elif self.legacy_package_manager_name is Constants.APT:
-                    output = ''
+                    if cmd.find('sudo apt-get install ubuntu-advantage-tools -y') > -1:
+                        code = 1
+                    else:
```

Comment:
nit: else is not required. The default value of output is '' so no need to set it.

Reply:
Done